docs/use/vscode.md

```diff
@@ -564,18 +564,32 @@ When you are following another participant, Live Share provides you more informa
 
 Live Share provides several additional settings to customize these audio cues.
 
-- `liveshare.accessibility.volume` - sets the volume of sounds and announcements.
-- `liveshare.accessibility.prosodyRate` - sets the prosody of announcements.
+- `liveshare.accessibility.soundsEnabled` - enables feedback sounds.
+- `liveshare.accessibility.soundVolume` - sets the volume of sounds.
+- `liveshare.accessibility.voiceEnabled` - enables spoken announcements.
+- `liveshare.accessibility.speechVolume` - sets the volume of announcements.
+- `liveshare.accessibility.speechRate` - sets the speech rate of announcements.
 - `liveshare.accessibility.voice` - sets the voice used for announcements. Currently supports 37 different voices, spanning genders, ages, and accents.
 
-Live Share also contributes a command `liveshare.accessibility.setOutputDevice` that allows you to change the output device used to play audio cues.
+Live Share also contributes several commands related to these features that can be bound to keyboard shortcuts.
 
-> This command is currently only available on Windows.
+- `Accessibility: Toggle Sounds Enabled` - toggles the `liveshare.accessibility.soundsEnabled` setting.
+- `Accessibility: Toggle Voice Enabled` - toggles the `liveshare.accessibility.voiceEnabled` setting.
+- `Accessibility: Announce Co-editor Location` - announces all co-editors' locations, regardless of if they're being followed.
+- `Accessibility: Set Output Device` - change the output device used to play audio cues. Note: this command is currently only available on Windows.
 
 ### Cursor tethering
 
 With accessibility features enabled, when following another participant Live Share will keep your cursor locked to the same position that the followed participant's cursor is at to make it easier to use the screen reader.
 
+### Braille Displays
+
+With the exception of cursor tethering, the accessibility features described above surface information audibly. We're in the process of exploring how to present this same information tactility to users with braille displays. If you'd like to learn more or be involved, please get in touch with us at vsls-feedback@microsoft.com.
+
+### Network Requirements
+
+Speech is generated by Azure Cognitive Services on demand. This means to receive speech feedback, you need to have an active internet connection and be able to reach the Cognitive Services endpoint - https://westus2.api.cognitive.microsoft.com.
+
```

Comment:
Is there a reason some networks wouldn't be able to reach this end-point? Seems interesting that we're calling this out? Is there any common mistake people make here?

Reply:
If you have a firewall set up, you may need to add this to an allow list.